### PR TITLE
Converted homepage from html css to jsx

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**
Added routes
Defined a folder structure for project

### CODE REVIEW

File: jersea/public/index.html [hunk 1]
```diff
@@ -15,6 +15,13 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="stylesheet" href="https://divine-ui.netlify.app/css/main.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/mockman-js@latest/dist/style.css"
+    />
+
```
Comment (anuj-kumary) [inline jersea/public/index.html]: Give here comments so that we know where you're using this link.

File: jersea/src/App.js [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
     </div>
   );
 }
 
+function App() {
+  return (
+    <>
+    <Routes>
+      <Route path='/' element={<Home/>} />
+      <Route path='/products'/>
```
Comment (anuj-kumary) [inline jersea/src/App.js:20]: In the products route, you didn't provide product components inside the element

File: jersea/src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            <div class="ecom-nav-action-btns">
+                <button class="ecom-nav-action-btn nav-btn-login">Login</button>
+                <div class="ecom-nav-action-btn badge">
+                    <i class="fas fa-shopping-cart ecom-nav-icon"></i>
+                    <span class="badge-number">6</span>
+                </div>
+             
+                <div class="ecom-nav-action-btn badge">
+                    <i class="fas fa-heart ecom-nav-icon"></i>
+                    <span class="badge-number">6</span>
```
Comment (anuj-kumary) [inline jersea/src/pages/Home/Home.jsx:24]: Change class to className

**Review by anuj-kumary — APPROVED** [review]

**Review by samyakshah3008 — APPROVED** [review]
Nice Folder structure  
Clean code for better review experience
Suggestion: You can add few comments in code as well later on

File: jersea/src/pages/Home/Home.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    color: var(--primary-color);
+    position: absolute;
+    bottom:1rem;
+    background-color: var(--text-color);
+}
+
+.grid-1{
+    width: 100%;
+    background-color:var(--primary-color);
+    height: 30rem;
```
Comment (samyakshah3008) [inline jersea/src/pages/Home/Home.css:84]: You can prefer giving max or min height for responsiveness

**Pull request merged**